packages/web/hooks/window/use-window-size.ts

```diff
@@ -19,7 +19,7 @@ export function useWindowSize(
   const [windowSize, setWindowSize] = useState<WindowSize>({
     width: 0,
     height: 0,
-    isMobile: false,
+    isMobile: true,
```

Comment:
I also added the `classNames`  function & removed the extra spaces  😄

Reply:
We default to desktop environments, why did you default to isMobile:true here?

Reply:
when you default to isMobile:false it will show the drawer on first render then transition out, but if this is a problem I can find a different way of dealing with this.

Reply:
Can we make this a parameter of `useWindowSize` then pass that in as `true` only from the `main.tsx` component? I'd like for us to remain desktop native.